rviz_rendering_tests/test/mesh_loader_test.cpp

```diff
@@ -72,7 +73,7 @@ TEST_F(MeshLoaderTestFixture, throws_reasonable_exception_for_missing_files) {
   std::string mesh_path = "package://rviz_rendering/ogre_media/MISSING.mesh";
   testing::internal::CaptureStderr();
 
-  auto mesh = rviz_rendering::loadMeshFromResource(mesh_path);
+  auto mesh = rviz_rendering::loadMeshFromResource(&this->retriever_, mesh_path);
 
   std::string output = testing::internal::GetCapturedStderr();
   ASSERT_THAT(output, HasSubstr("Error retrieving file"));
```

Comment:
```suggestion
  ASSERT_THAT(mesh, nullptr);
```

Reply:
I'm confused about this change, why are we not using the `output` variable anymore? If that's correct, then why not remove the line before it too?